rulebook.tex

```diff
@@ -1992,8 +1992,8 @@ \section{The Robotino System}
 may only occupy up to $25\%$ of this additional \SI{0.14}{\metre}
 (Robotino 2) or \SI{0.10}{\metre} (Robotino 3) wide ring around the
 robot. The only additional actuator allowed is one gripping device for
-workpieces which can be the original or a modified one. It also must
-be within the same diameter of the other additional hardware. The
+workpieces which can be the original or a modified one. In the rest position and while driving it also mustbe within the same diameter of the other additional hardware.
```

Comment:
typo: mustbe -> must be

Also, please keep the lines shorter than 80 characters, as in the surrounding paragraph.